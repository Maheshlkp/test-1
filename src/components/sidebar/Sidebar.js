import React from "react";
import { Link } from "react-router-dom";

export const Sidebar = () => {
  return (
    <nav>
      <ul>
        <li>
          <Link to="/">Add</Link>
        </li>
        <li>
          <Link to="/result">Result</Link>
        </li>
      </ul>
    </nav>
  );
};
