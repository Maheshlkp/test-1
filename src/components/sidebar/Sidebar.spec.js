import React from 'react';
import {shallow} from 'enzyme';
import {Sidebar} from './Sidebar';

describe('Test Sidebar', () => {

    it('should render correctly', () => {
        const wrapper = shallow(<Sidebar />);
        expect(wrapper.getElements()).toMatchSnapshot();
    })
});