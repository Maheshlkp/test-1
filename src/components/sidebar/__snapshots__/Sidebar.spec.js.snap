// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Sidebar should render correctly 1`] = `
Array [
  <nav>
    <ul>
      <li>
        <Link
          to="/"
        >
          Add
        </Link>
      </li>
      <li>
        <Link
          to="/result"
        >
          Result
        </Link>
      </li>
    </ul>
  </nav>,
]
`;
